Makefile-tests.am

```diff
@@ -238,14 +239,34 @@ EXTRA_DIST += \
 tests/libreaddir-rand.so: Makefile
 	$(AM_V_GEN) ln -fns ../.libs/libreaddir-rand.so tests
 ALL_LOCAL_RULES += tests/libreaddir-rand.so
-CLEANFILES += tests/libreaddir-rand.so tests/ostree-symlink-stamp tests/ostree
+CLEANFILES += tests/libreaddir-rand.so tests/ostree-symlink-stamp \
+		tests/ostree-prepare-root-symlink-stamp tests/ostree-remount-symlink-stamp \
+		tests/rofiles-fuse-symlink-stamp tests/ostree
 
 tests/ostree-symlink-stamp: Makefile
 	@set -e; \
 	real_bin=`cd $(top_builddir) && ./libtool --mode=execute echo ostree`; \
 	ln -sf "$${real_bin}" tests/ostree; \
 	touch $@
 
+tests/rofiles-fuse-symlink-stamp: Makefile
```

Comment:
I could make for the new 3 ones, but the rule for ostree is using libtool, while the new ones point directly to the binary file